source/guide_soju.rst

```diff
@@ -0,0 +1,139 @@
+.. highlight:: console
+
+.. author:: apoxa <https://github.com/apoxa>
+
+.. tag:: lang-go
+.. tag:: irc
+.. tag:: bouncer
+
+.. spelling::
+    Soju
+    soju
+    upstreams
+
+##########
+Soju
+##########
+
+.. tag_list::
+
+soju_ is a user-friendly IRC bouncer which supports multiple users, multiple clients for a single user with backlog synchronization and multiple upstreams via a single IRC connection to the bouncer.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Firewall Ports <basics-ports>`
+  * :manual:`supervisord <daemons-supervisord>`
+
+License
+=======
+
+All relevant legal information can be found here:
+
+  * https://git.sr.ht/~emersion/soju/tree/master/item/LICENSE
+
+Prerequisites
+=============
+
+.. include:: includes/open-port.rst
+
+Installation
+============
+
+Clone the source into your home directory and enter the directory:
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ git clone https://git.sr.ht/~emersion/soju
+ [isabell@stardust ~]$ cd ~/soju
+ [isabell@stardust soju]$
+
+Compile and install soju to your home directory:
+
+.. code-block:: console
+
+ [isabell@stardust soju]$ make -W doc/soju.1
+ [...]
+ [isabell@stardust soju]$ PREFIX="/.local" DESTDIR="$HOME" make install
+ [...]
+ [isabell@stardust soju]$
+
+Configuration
+=============
+
+Configure soju
+--------------
+
+Save the following as your soju configuration in ``~/etc/soju/config``.
+
+.. code-block:: ini
+
+ listen ircs://0.0.0.0:40132
+ tls /home/isabell/etc/certificates/isabell.uber.space.crt /home/isabell/etc/certificates/isabell.uber.space.key
+ hostname isabell.uber.space
+ db sqlite3 /home/isabell/var/lib/soju/soju.db
+
+.. warning:: Replace `ìsabell` with your username and the port the one
+
+Setup daemon
+------------
+Create ``~/etc/services.d/soju.ini`` with the following content:
+
+.. code-block:: ini
+
+ [program:soju]
+ command=%(ENV_HOME)s/.local/bin/soju -config %(ENV_HOME)s/etc/soju/config
+ autostart=yes
+ autorestart=yes
+
+.. include:: includes/supervisord.rst
+
+If it's not in state RUNNING, check your configuration.
+
+Create a user
+-------------
+.. note:: Replace ``isabell`` with your username!
+
+.. code-block:: console
+
+ [isabell@stardust ~]$ ./.local/bin/sojuctl -config $HOME/etc/soju/config create-user isabell -admin
+ Password:
+ [isabell@stardust ~]$
+
+It will prompt you for a password now.
+
+You can add additional users with the same command or later via the BouncerServ_ builtin service.
+
+Connect your client
+-------------------
+
+Add a new server in your local IRC client:
+
+* Hostname: `<username>.uber.space`
+* Port: your port
+* Username: soju nickname
```

Comment:
it should be better referenced, that these nickname/passwords are ones set before